client/main.lua

```diff
@@ -21,9 +21,26 @@ local function PlayATMAnimation(animation)
     end
 end
 
+local listen = false
+local function Listen4Control()
+    CreateThread(function()
+        listen = true
+        while listen do
+            if IsControlJustPressed(0, 38) then -- E
+                    exports["qb-core"]:KeyPressed()
+                    TriggerServerEvent("qb-atms:server:enteratm")
+                listen = false
+                break
+            end
+            Wait(1)
+        end
+    end)
+end
```

Comment:
```suggestion
local listen = false
local function Listen4Control()
    listen = true

    CreateThread(function()
        while listen do
            if IsControlJustPressed(0, 38) then -- E
                exports["qb-core"]:KeyPressed()
                TriggerServerEvent("qb-atms:server:enteratm")
                listen = false
                break
            end
            Wait(1)
        end
    end)
end
```
Fix linting / spacing